.buildkite/bk.integration.pipeline.yml

```diff
@@ -38,115 +38,45 @@ steps:
       - package-it
       - integration-ess
     steps:
-      - label: "Default"
-        key: stateful-ubuntu-default-sudo
+      - label: "x86:Group: {{matrix}}"
```

Comment:
nit: more accurate to call this `x64`? Or `x86-x64`?

Reply:
Let's go with x86_64, since I think we use that term more consistently in other places